nyxl-theme/src/css/_custom.scss

```diff
@@ -0,0 +1,416 @@
+@import "portlet_decorator";
+
+@import "portlet_hello_world";
+
+@import "header";
+
+body {
+	background-color: $bg-color;
+	font-family: $body-font;
+}
+
+a {
+	transition: 0.3s ease-in-out;
+	// white-space: nowrap;
+}
+
+.container-fluid-1280 .row,
+.portlet {
+	margin: 0;
+}
+
+.btn, .btn-default {
+	@include btnBackground($lightblue-color);
+	font-family: $link-text !important;
+	font-size: 12px;
+	font-weight: bold;
+	transition: $transition;
+
+	@include textStyle(2px !important, uppercase !important);
+}
+
+.btn:hover, .btn:focus {
+	@include btnBackground($brightblue-color);
+	text-shadow: $hard-shadow;
+}
+
+.text-default {
+	color: $owgray-color;
+}
+
+/* Animated Links on Hover */
+
+.portlet-blogs h2 > a::after,
+#_com_liferay_wiki_web_portlet_WikiPortlet_navTag li::after {
+	background: #FFF;
+	content: '';
+	display: block;
+	height: 2px;
+	transition: width 0.3s;
+	width: 0;
+}
+
+/* Red Underline*/
+.portlet-blogs h2 > a::after {
+	background: $nyred-color;
+}
+
+/* Lightblue Underline */
+#_com_liferay_wiki_web_portlet_WikiPortlet_navTag li::after {
+	background: $lightblue-color;
+}
+
+.portlet-blogs h2 > a:hover::after,
+#_com_liferay_wiki_web_portlet_WikiPortlet_navTag li:hover::after {
+	width: 100%;
+}
+
+/* Non animated style when active */
+#_com_liferay_wiki_web_portlet_WikiPortlet_navTag li.active:hover::after {
+	width: 0%;
+}
+
+/* Text Styles */
+
+a {
+	font-family: $link-text;
+}
+
+.portlet-title-text,
+thead,
+.header-title,
+h1, h2, h3, h4, h5, h6,
+.h1, .h2, .h3, .h4, .h5, .h6 {
```

Comment:
If we are going to put all the CSS selectors on a new line, we should be consistent.